transformations.html

```diff
@@ -606,8 +670,20 @@ <h2>Modules</h2>
 import Maybe ( Maybe(..) )    -- Maybe, Just, Nothing
  </code></pre>
 
-<pre><code class="roc haskell">
-TBC
+<pre><code class="roc coffeescript">
+interface MyModule
+    exposes [ foo, blah ] # Roc does not have "expose everything" syntax
+    imports
+        [
+            # Qualified imports
+            List as L, # L.map, L.walk
+            
+            # Open imports (Roc does not have "expose everything" syntax)
+            List.{ map, walk }
+            
+            # Imports the Result module and also the Result type
+            Result.{ Result }
```

Comment:
are there commas after?

```suggestion
            List.{ map, walk },
            
            # Imports the Result module and also the Result type
            Result.{ Result },
```

Reply:
Yes - thanks!